test/integration/auth_test.go

```diff
@@ -938,14 +857,8 @@ func TestNamespaceAuthorization(t *testing.T) {
 	framework.DeleteAllEtcdKeys()
 
 	// This file has alice and bob in it.
-	etcdStorage, err := framework.NewEtcdStorage()
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	expEtcdStorage, err := framework.NewExtensionsEtcdStorage(nil)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
+	etcdStorage := framework.NewEtcdStorage()
```

Comment:
It seems that this one (and the following lines, e.g. the whole storageDestinations-related code) is no longer needed.
Can you please remove it?